modules/ROOT/pages/maven-reference.adoc

```diff
@@ -45,8 +45,39 @@ In this case, it is a _user_ `settings.xml` file that is relevant only for a spe
 
 On a system with both global and user `settings.xml` files, the settings from both files become merged. In case there are duplicate settings, the user's settings take precedence over the global settings.
 
+== Mule Repositories
+
+When you create a new Mule project from Anypoint Studio, the minimum required Mule repositories are automatically added to the `pom.xml` file. If you are creating the `pom.xml` file manually, add the following repositories:
+
+[source,xml,linenums]
+----
+<repository>
+    <id>anypoint-exchange</id>
+    <name>Anypoint Exchange</name>
+    <url>https://maven.anypoint.mulesoft.com/api/v1/maven</url>
+    <layout>default</layout>
+</repository>
+<repository>
+    <id>mulesoft-releases</id>
+    <name>MuleSoft Releases Repository</name>
+    <url>https://repository.mulesoft.org/releases/</url>
+    <layout>default</layout>
+</repository>
+<pluginRepository>
+    <id>mulesoft-releases</id>
+    <name>mulesoft release repository</name>
+    <layout>default</layout>
+    <url>https://repository.mulesoft.org/releases/</url>
+    <snapshots>
+        <enabled>false</enabled>
+    </snapshots>
+</pluginRepository>
+----
+
 == Useful Commands
 
+The following set of commands enables you to retrieve information related to your Maven installation, as well as perform certain project-related tasks.
```

Comment:
"Related to" or "about"? I think "about" is better.
Also, I'd change the supplementary clause to a verbal phrase joined to the rest of the sentence by a conjunction:
"The following set of commands enables you to retrieve information about your Maven installation and to perform project-related tasks."
"Certain" is superfluous.